pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.microsoft.azure</groupId>
     <artifactId>spring-data-documentdb</artifactId>
-    <version>0.1.4-SNAPSHOT</version>
+    <version>0.1.4</version>
```

Comment:
pls also help to update readme if there's. or we can have a maven version badge in readme, like azure-spring-boot readme.md